lmcache/v1/storage_backend/abstract_backend.py

```diff
@@ -77,8 +77,9 @@ def batched_submit_put_task(
         """
         An async function to put the MemoryObj into the storage backend.
 
-        :param List[CacheEngineKey] keys: The keys of the MemoryObjs.
+        :param Sequence[CacheEngineKey] keys: The keys of the MemoryObjs.
```

Comment:
Any reason changed list to sequence? for the string performance?

Reply:
No actually its sequence only, the description was wrong here which I just updated.